lib/cdo/data/google_sheet_to_csv.rb

```diff
@@ -15,9 +14,9 @@ def initialize(path)
   end
 
   def up_to_date?
-    @file ||= (@@gdrive ||= Google::Drive.new).file(@gsheet_path)
+    @file ||= (@@gdrive ||= Google::Drive.new).file_by_id(@gsheet_id)
     unless @file
-      ChatClient.log "Google Drive file <b>#{@gsheet_path}</b> not found.", color: 'red', notify: 1
+      CDO.log.info "Google Drive file with id: #{@gsheet_id} not found."
```

Comment:
What's the motivation to stop logging results to slack?

Also, I can never remember; where does `CDO.log` output end up when the job is being run by cron? Honeybadger?

Reply:
My thinking here is that the purpose of this module is to provide a simplified wrapper for interacting with Google Drive and Google Sheets. If we have a build component (like a rake task or the `GSheetToCSV` class that uses this module, that component is welcome to rescue errors and log them to an `#infra-` Slack channel.

Also, I disliked how noisy it was.

`CDO.log.info` is out standard logger and gets streamed to CloudWatch.

Reply:
That makes sense; but, given that in practice this file is primarily used by the gsheet sync cronjob, we should also include a change to apply the error-resucing and `#infra-staging`-logging to the associated rake task. I do think we want this to be noisy when it's broken, so that we know to fix it.

Might make sense to split this off into a separate PR